ucac/scripts/run_local_tests.sh

```diff
@@ -14,8 +14,8 @@ function cleanup {
 
 trap cleanup EXIT
 cp ../lndr-backend/data/lndr-server.config.test ../lndr-backend/data/lndr-server.config
-npm run ganache &
-ganache_pid=$!
+
+ganache_pid=`npm run ganache`
```

Comment:
Clever, moving this to the npm package manifest. Does tidy it up a bit.

Reply:
And the earlier version of this script was broken since I was grabbing the pid of `npm` and not the `ganache-cli` process it started. I have to echo the pid from within the `npm` command out to my `run_local_tests.sh` so that the test script knows what pid to kill later on.